main.js

```diff
@@ -13,23 +14,22 @@ gameBoard.on('click', function(){
 startButton.on('click', function(e) {
   e.preventDefault();
   let playersArray = [
-    document.querySelector('.js-p1-name-input').value, 
-    document.querySelector('.js-p2-name-input').value, 
-    document.querySelector('.js-p3-name-input').value
+    new Player(document.querySelector('.js-p1-name-input').value).playerName, 
+    new Player(document.querySelector('.js-p2-name-input').value).playerName, 
+    new Player(document.querySelector('.js-p3-name-input').value).playerName
   ]
-  let game = new Game(playersArray); 
-
-  // console.log({game})
-  // console.log({playersArray})
-
+  console.log(playersArray)
+  let game = new Game(playersArray);
+  let round = new Round();
   domUpdates.updatePlayerNames(playersArray);
-  
-  game.createNewRound();
-
   $('.js-login-page').hide();
   $('.main-page').show();
-
+  console.log(game)
 });
 
+
+
+
 // converting camelCase category names to spaces: 
-// https://stackoverflow.com/questions/7225407/convert-camelcasetext-to-sentence-case-text
+// https://stackoverflow.com/questions/7225407/
+// convert-camelcasetext-to-sentence-case-text
```

Comment:
This type of information should go in the description of an issue rather than cluttering up the diffs on your PRs

Reply:
(Same with console.logs, make sure those get taken out before you commit and push)